File: tests/benchmarks/cases/sort_array/c/benchmark_case.c
#include "benchmark_case.h"
#include "benchmark_support.h"

static ZrBenchInt zr_bench_case_sort_array_run(int scale) {
    return zr_bench_run_sort_array(scale);
}

const ZrBenchCaseDescriptor zr_bench_case_descriptor_sort_array = {
        "sort_array",
        "BENCH_SORT_ARRAY_PASS",
        zr_bench_case_sort_array_run
};

File: tests/benchmarks/cases/sort_array/java/benchmark_case.java
final class SortArrayCase {
    static final String NAME = "sort_array";
    static final String PASS_BANNER = "BENCH_SORT_ARRAY_PASS";

    private SortArrayCase() {}

    static long run(int scale) {
        return BenchmarkSupport.sortArray(scale);
    }
}
